### PR TITLE
Docs for re-register, and node+delegator

### PR DESCRIPTION
**Pull request by joshuahannan**
Closes: https://github.com/onflow/flow-core-contracts/issues/73

## Description

Adds clarification that a node or delegator can re-register a different one, but only if they have withdrawn ALL their tokens beforehand.

Also adds section in each section saying that there can be a node and delegator for the same account.

For contributor use:

- [x] Targeted PR against `master` branch
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [x] Updated relevant documentation 
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [x] Added appropriate labels

### CODE REVIEW

File: docs/content/staking/locked-delegation-guide.mdx [hunk 1]
```diff
@@ -146,3 +150,8 @@ If a token holder wants to create a second delegation relationship, they must cr
 and transfer their locked tokens to the new account by requesting authorization from the token admin.
 
 We plan on adding functionality in the future to allow accounts to delegate to multiple nodes.
+
+## Stake from the Same Account
+
+An account who registers a delegator with locked tokens can also register a delegator in the same account.
```
Comment (psiemens) [inline docs/content/staking/locked-delegation-guide.mdx:156]:
```suggestion
An account that registers as a delegator with locked FLOW can also register a staked node in the same account.
```

File: docs/content/staking/locked-delegation-guide.mdx [hunk 2]
```diff
@@ -146,3 +150,8 @@ If a token holder wants to create a second delegation relationship, they must cr
 and transfer their locked tokens to the new account by requesting authorization from the token admin.
 
 We plan on adding functionality in the future to allow accounts to delegate to multiple nodes.
+
+## Stake from the Same Account
+
+An account who registers a delegator with locked tokens can also register a delegator in the same account. 
+See the Locked Staking Guide for how to register a node.
```
Comment (psiemens) [inline docs/content/staking/locked-delegation-guide.mdx:157]:
```suggestion
See the [Locked Staking Guide](../locked-staking-guide) for how to register a node.
```

File: docs/content/staking/locked-staking-guide.mdx [hunk 3]
```diff
@@ -38,10 +38,14 @@ This transaction registers the account as a staker with the specified node infor
 and attaches a `NodeStakerProxy` capability to the `TokenHolder` resource. 
 This capability can later be used to perform staking actions.
 
-Once an account has registered as a staker with one node, they cannot register with any others. 
+Once an account has registered as a staker with one node, they cannot register any additional.
```
Comment (psiemens) [inline docs/content/staking/locked-staking-guide.mdx:41]:
```suggestion
Once an account has registered as a staker with one node, they cannot register any additional nodes.
```

File: docs/content/staking/locked-staking-guide.mdx [hunk 4]
```diff
@@ -159,3 +163,8 @@ If a token holder wants to create a second staking relationship, they must creat
 and transfer their locked tokens to the new account by requesting authorization from the token admin.
 
 We plan on adding functionality in the future to allow accounts to stake to multiple nodes.
+
+## Delegate from the Same Account
+
+An account who registers a node with locked tokens can also register a delegator in the same account.
```
Comment (psiemens) [inline docs/content/staking/locked-staking-guide.mdx:169]:
```suggestion
An account that registers a node with locked FLOW can also register as delegator from the same account. 
```

File: docs/content/staking/locked-staking-guide.mdx [hunk 5]
```diff
@@ -159,3 +163,8 @@ If a token holder wants to create a second staking relationship, they must creat
 and transfer their locked tokens to the new account by requesting authorization from the token admin.
 
 We plan on adding functionality in the future to allow accounts to stake to multiple nodes.
+
+## Delegate from the Same Account
+
+An account who registers a node with locked tokens can also register a delegator in the same account. 
+See the Locked Delegation Guide for how to register a delegator.
```
Comment (psiemens) [inline docs/content/staking/locked-staking-guide.mdx:170]:
```suggestion
See the [Locked Delegation Guide](../locked-delegation-guide) for how to register a delegator.
```

**Comment by joshuahannan** [comment]
@psiemens I fixed the changes from your comments. Can I get an approval?

**Review by psiemens — APPROVED** [review]

**Pull request merged**